pyalgotrading/constants.py

```diff
@@ -177,3 +177,8 @@ class PlotType(Enum):
     RENKO = 'Renko'
     HEIKINASHI = 'HeikinAshi'
     QUANDL_JAPANESE = 'Quandl JAPANESE'
+
+
+MESSAGE_REALTRADING_FORBIDDEN = 'Forbidden. As per the new policy, directly running a strategy in RT mode is prohibited. ' \
```

Comment:
suggested alternative description:

'Forbidden. This strategy is not approved for RT mode. Email us on support@algobulls.com to get your strategy approved. The AlgoBulls support team will guide you with the approval process. Once your strategy is approved, you can run it for RT directly from your AlgoBulls account.'

Reply:
@pushpakdagade

Reply:
suggestion taken; updated the constant